testproject/Assets/Scenes/SampleScene.unity

```diff
@@ -534,17 +533,10 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: e96cb6065543e43c4a752faaa1468eb1, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  TransformAuthority: 0
-  FixedSendsPerSecond: 30
-  InterpolatePosition: 1
-  SnapDistance: 10
-  InterpolateServer: 1
-  MinMeters: 0.15
-  MinDegrees: 1.5
-  MinSize: 0.15
+  Authority: 0
   Channel: 10
-  m_UseLocal:
-    m_InternalValue: 1
+  InLocalSpace: 0
```

Comment:
should be 1

Reply:
oh, that one was that nested scale cube thingy, right. I just wasn't sure.